sklearn/hmm.py

```diff
@@ -623,23 +626,27 @@ class GaussianHMM(_BaseHMM):
 
     Parameters
     ----------
+
     n_components : int
         Number of states.
 
-    ``_covariance_type`` : string
+    covariance_type : string, optional, default "diag"
         String describing the type of covariance parameters to
+ 
+    algorithm : string, one of the decoder_algorithms, default "viterbi"
+        Decoder algorithm to be used.
         use.  Must be one of 'spherical', 'tied', 'diag', 'full'.
-        Defaults to 'diag'.
-
-    Attributes
-    ----------
-    ``_covariance_type`` : string
```

Comment:
The function signature had only `covariance_type` that was why the script did not throw any further errors... You were right the old one was just wrong...